files/default/neutron-ha-tool.py

```diff
@@ -219,18 +222,18 @@ def run(args):
     elif args.l3_agent_migrate:
         LOG.info("Performing L3 Agent Migration for Offline L3 Agents")
         errors = retry_with_backoff(l3_agent_migrate, args)(
-            qclient, args.noop, args.now)
+            qclient, args.noop, args.now, args.wait_for_ports)
```

Comment:
@aspiers Are you ok if I address this with https://github.com/SUSE-Cloud/cookbook-openstack-network/pull/8 ?